packages/dashboard/src/components/Wizard/FullParentWizard.types.ts

```diff
@@ -0,0 +1,18 @@
+import { IWizardProps, IWizardStyles } from './Wizard.types';
+import { IStyle, ITheme } from 'office-ui-fabric-react/lib/Styling';
+
+export interface IFullParentWizardProps extends IWizardProps {

```

Comment:
>extends IWizardProps [](start = 40, length = 20)

IWizardProps also has "styles" member.  Hence we included it as a property within SetupWizardProps and PanelWizardProps, instead of extends.

Can you also do the same?